app/helpers/reactjs_helper.rb

```diff
@@ -45,10 +50,20 @@ def js_tags_for(requested_plugins)
     end
   end
 
-  def js_tags_for_global_files(requested_plugins)
+  def global_js_tags(requested_plugins)
     requested_plugins.map do |plugin|
       plugin[:files].map do |file|
-        javascript_include_tag(*webpack_asset_paths(plugin[:id].to_s + ":#{file}", :extension => 'js'), :defer => "defer")
+        javascript_include_tag(*webpack_asset_paths("#{plugin[:id]}:#{file}", :extension => 'js'), :defer => "defer")
+      end
+    end
+  end
+
+  def global_css_tags(requested_plugins)
+    requested_plugins.map do |plugin|
+      plugin[:files].map do |file|
+        stylesheet_link_tag(*webpack_asset_paths("#{plugin[:id]}:#{file}", :extension => 'css'))
+      rescue Webpack::Rails::Manifest::EntryPointMissingError
```

Comment:
@amirfefer this is never triggered. As according to https://www.rubydoc.info/gems/webpack-rails/0.9.7/Webpack/Rails/Manifest#asset_paths-class_method the exception is raised only if the chunk doesn't exist in the manifest and the manifest seems to have names withouth the extenstion.
This is what my `Webpack::Rails::Manifest.send(:manifest)["assetsByChunkName"]` looks like (`foreman_statistics` has two global files one - fills - with css and one without - globals).
```ruby
{"foreman-vendor"=>
  ["foreman-vendor.bundle-v4.15.1-development-e5b8494c82f561b65232.css",
   "foreman-vendor.bundle-v4.15.1-development-e5b8494c82f561b65232.js",
   "foreman-vendor.bundle-v4.15.1-development-e5b8494c82f561b65232.css.map",
   "foreman-vendor.bundle-v4.15.1-development-e5b8494c82f561b65232.js.map"],
 "bundle"=>["bundle.js", "bundle.css"],
 "foreman_statistics:fills"=> ["foreman_statistics:fills.js", "foreman_statistics:fills.css"],
 "foreman_statistics"=>"foreman_statistics.js",
 "foreman_statistics:globals"=>"foreman_statistics:globals.js",
 "vendor"=>"vendor.js"}
```
So `webpack_asset_paths("foreman_statistics:globals", extension: 'css')` return empty array (get the path without issues and then filter out all of them.
And thus my concern was not valid in the first place because `stylesheet_link_tag([]) => ''`

Can you please drop the rescue? :)